sycl/test/esimd/intrins_trans.cpp

```diff
@@ -1,8 +1,12 @@
-// RUN: %clangxx -O0 -fsycl -fno-sycl-esimd-force-stateless-mem -fsycl-device-only -Xclang -emit-llvm -Xclang -no-opaque-pointers %s -o %t
+// RUN: %clangxx -O0 -fsycl -fsycl-device-only -fno-sycl-esimd-force-stateless-mem -Xclang -emit-llvm %s -o %t
```

Comment:
this (and all the tests) are just a file rename from _opaque with a possible -opaque-pointer flag removal, you don't have to review the test contents itself, they are all already in-tree.